simapp/v2/simdv2/cmd/testnet.go

```diff
@@ -504,3 +510,13 @@ func writeFile(name, dir string, contents []byte) error {
 
 	return os.WriteFile(file, contents, 0o600)
 }
+
+func getBankV2GenesisFromV1(v1GenesisState banktypes.GenesisState) bankv2types.GenesisState {
```

Comment:
let's add a comment on the rationale of doing this (basically, to be deleted after genesis migration command addition)